cmake/nanobind-config.cmake

```diff
@@ -120,6 +120,7 @@ function (nanobind_build_library TARGET_NAME)
     ${NB_DIR}/src/trampoline.cpp
     ${NB_DIR}/src/implicit.cpp
   )
+  add_library(nanobind::nanobind ALIAS ${TARGET_NAME})
```

Comment:
The interface is documented though https://nanobind.readthedocs.io/en/latest/api_cmake.html#low-level-interface

I just think it could be improved for those who want more control.